### PR TITLE
DEV-3889-weareunique subject

### PR DESCRIPTION
**Pull request by zainabdnaya**

### CODE REVIEW

**Review by Hamzaelkhatri — APPROVED** [review]
LGTM!

File: subjects/weareunique/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+## Weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the amount of characters that are not included in both without doubles.
```
Comment (MSilva95) [inline subjects/weareunique/README.md:5]:
```
Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.

- If there are no unique characters or if any of the strings is empty, return -1
```

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/weareunique/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+)
+
+func main() {
+
+	arr := [][]string{
+		{"foo", "boo"},
+		{"",""},
+		{"abc", "def"},
+		{"hello", "yoall"},
+		{"everyone", ""},
```
Comment (MSilva95) [inline subjects/weareunique/README.md:38]: Your solution returns 6 here

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/weareunique/README.md (hunk 2 above)
Comment (zainabdnaya) [reply subjects/weareunique/README.md:38]: I forgot to push the right solution in  the test repository but now it's good

File: subjects/weareunique/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+## Weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.
+- If there is no unique charachters return 0.
+- If both strings are empty  or one of them return -1.
```
Comment (MSilva95) [inline subjects/weareunique/README.md:7]: We should make this only when both are empty, when only one is empty it should return the unique chars...
so this: {"everyone", ""} and {"","everyone"} will return 6

**Review by MSilva95 — CHANGES_REQUESTED** [review]

File: subjects/weareunique/README.md (hunk 3 above)
Comment (zainabdnaya) [reply subjects/weareunique/README.md]: okay , it's done

**Review by jotapero — APPROVED** [review]
Looks Good :alien:

File: subjects/weareunique/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+## weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.
```
Comment (tiagocollot) [inline subjects/weareunique/README.md:5]: - The word strings should be formatted --> `strings`. 
-  Add missing space in between line 5 and line 6.

File: subjects/weareunique/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
```
Comment (tiagocollot) [inline subjects/weareunique/README.md:25]: Add missing import piscine.

File: subjects/weareunique/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+)
+
+func main() {
```
Comment (tiagocollot) [inline subjects/weareunique/README.md:28]: -  'func main' needs to be formatted correctly in order to match the subjects structure like we have been doing in the previous exercises.

File: subjects/weareunique/README.md (hunk 4 above)
Comment (tiagocollot) [inline subjects/weareunique/README.md:5]: This a suggestion:  "...without repeated characters" instead of "...without doubles."

**Review by tiagocollot — CHANGES_REQUESTED** [review]

File: subjects/weareunique/README.md (hunk 4 above)
Comment (jotapero) [reply subjects/weareunique/README.md:5]: agreed. Fixed

File: subjects/weareunique/README.md (hunk 6 above)
Comment (jotapero) [reply subjects/weareunique/README.md:28]: Agreed. Fixed

File: subjects/weareunique/README.md (hunk 4 above)
Comment (jotapero) [reply subjects/weareunique/README.md]: fixed

File: subjects/weareunique/README.md (hunk 5 above)
Comment (jotapero) [reply subjects/weareunique/README.md:26]: fixed

**Review by tiagocollot — APPROVED** [review]
LGTM! ⚡

**Pull request merged**